packages/react-native-win32/.flowconfig

```diff
@@ -39,10 +46,6 @@
 <PROJECT_ROOT>/RNTester/js/components/ListExampleShared.js
 <PROJECT_ROOT>/RNTester/js/components/RNTesterExampleFilter.js
 
-; These examples currently uses mac dynamic colors
-<PROJECT_ROOT>/RNTester/js/ActivityIndicatorExample.js
```

Comment:
DarkModeExample doesn't exist in the repo and the top doesn't seem to have issues? Removed the suppression.